Packs/AssetSearch/IncidentFields/incidentfields.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
rename file to `incidentfield-assetlist.json` and keep only the json here, no need to be in a list of incident fields